examples/04_glm_first_level/plot_first_level_details.py

```diff
@@ -198,7 +198,7 @@ def plot_contrast(first_level_model):
     design_matrix = first_level_model.design_matrices_[0]
     # Call the contrast specification within the function
     contrasts = make_localizer_contrasts(design_matrix)
-    plt.figure(figsize=(11, 3))
+    plt.figure(figsize=(20, 4))
```

Comment:
Why this change ?

Reply:
to avoid the isssue of #4228 where subplot title overlap with each other and with the color bar